charts/ocis/templates/ocm/deployment.yaml

```diff
@@ -0,0 +1,121 @@
+{{ if .Values.features.ocm.enabled }}
+{{- include "ocis.basicServiceTemplates" (dict "scope" . "appName" "appNameOcm" "appNameSuffix" "") -}}
+apiVersion: apps/v1
+kind: Deployment
+{{ include "ocis.metadata" . }}
+spec:
+  {{- include "ocis.selector" . | nindent 2 }}
+  replicas: 1 # OCM service cannot be scaled right now
+  {{- include "ocis.deploymentStrategy" . | nindent 2 }}
+  template:
+    {{- include "ocis.templateMetadata" (dict "scope" $ "configCheck" false) | nindent 4 }}
+    spec:
+      {{- include "ocis.affinity" $ | nindent 6 }}
+      {{- include "ocis.securityContextAndtopologySpreadConstraints" . | nindent 6 }}
+      {{- include "ocis.priorityClassName" $.priorityClassName | nindent 6 }}
+      {{- include "ocis.hostAliases" $ | nindent 6 }}
+      nodeSelector: {{ toYaml $.nodeSelector | nindent 8 }}
+      containers:
+        - name: {{ .appName }}
+          {{- include "ocis.image" $ | nindent 10 }}
+          command: ["ocis"]
+          args: ["ocm", "server"]
+          {{- include "ocis.containerSecurityContext" . | nindent 10 }}
+          env:
+            {{- include "ocis.serviceRegistry" . | nindent 12 }}
+            {{- include "ocis.cors" . |nindent 12 }}
+            {{- include "ocis.events" . | nindent 12 }}
+
+            - name: OCM_LOG_COLOR
+              value: {{ .Values.logging.color | quote }}
+            - name: OCM_LOG_LEVEL
+              value: {{ .Values.logging.level | quote }}
+            - name: OCM_LOG_PRETTY
+              value: {{ .Values.logging.pretty | quote }}
+
+            # Tracing not yet implemented: https://github.com/owncloud/ocis/issues/6176
+            - name: OCM_TRACING_ENABLED
+              value: "{{ .Values.tracing.enabled }}"
+            - name: OCM_TRACING_TYPE
+              value: {{ .Values.tracing.type | quote }}
+            - name: OCM_TRACING_ENDPOINT
+              value: {{ .Values.tracing.endpoint | quote }}
+            - name: OCM_TRACING_COLLECTOR
+              value: {{ .Values.tracing.collector | quote }}
+
+            - name: OCM_DEBUG_PPROF
+              value: {{ .Values.debug.profiling | quote }}
+
+            - name: OCM_DEBUG_ADDR
+              value: 0.0.0.0:9281
+
+            - name: OCM_HTTP_ADDR
+              value: 0.0.0.0:9280
+
+            - name: OCM_GRPC_ADDR
+              value: 0.0.0.0:9282
+
+            - name: OCIS_URL
+              value: "https://{{ .Values.externalDomain }}"
```

Comment:
IIRC engineering asked me to add this. @butonic could you please state on this?

Reply:
OCIS_URL is pulled in from the common config and used as the `provider_domain`. I'll create an ocis issue because I think there are more places that use the common configs ...

Reply:
added a comment in the code: https://github.com/owncloud/ocis-charts/pull/701/commits/cb8ef6ae0e901105f03a5817b6aae7cf9c34663b